### ISSUE TITLE
[Regression] Inputs with options (dropdowns) are completely missing from the parameter list

### ISSUE THREAD

**Issue by RealWenzhangGe** [akumidv/tradingview-assistant-chrome-extension#304]
Dropdown menus (input.int with options argument) used to work in previous versions but are now undetected. This is a regression.

Current Status:

<img width="1586" height="1586" alt="Image" src="https://github.com/user-attachments/assets/366467e3-e1e7-4437-8e7c-51f147a93166" />

<img width="2046" height="916" alt="Image" src="https://github.com/user-attachments/assets/ded35ee9-740d-4e3b-b0f8-f31bfb8d214e" />

✅ What works (Detected):

Standard integer inputs (defined by min/max/step).

Boolean checkboxes (input.bool).

Float inputs.

❌ What is missing (Not Detected):

Any input using the options parameter.

Example: input.int(..., options=[10, 20, 50]).

These inputs are visible in the TradingView chart settings but do not appear in the Assistant's parameter list for backtesting.

Steps to Reproduce:

Apply the script below to a chart.

Open the extension/tool.

Observe that Lookback Period (N) is missing from the list, while Hold Bars (M) and Enable % Stop Loss are present.

Reproduction Code:

//@version=5
strategy("Bug Reproduction", overlay=true)

// --- ❌ MISSING IN TOOL ---
// This input uses 'options' and does NOT show up in the assistant.
n_lookback = input.int(20, title="Lookback Period (N)", options=[10, 20, 50, 100])

// --- ✅ PRESENT IN TOOL ---
// These inputs work fine.
m_exit_bars = input.int(10, title="Hold Bars (M)")
use_stop_loss = input.bool(true, title="Enable % Stop Loss")

**Comment by Satanbear** [comment]
I have the same problem, only with strings, I only came up with such an inconvenient solution
`tpTypeInt           = input.int    (3,              ' 𝐓𝐚𝐤𝐞 𝐏𝐫𝐨𝐟𝐢𝐭 INT',                     group=groupset, inline='typetp', minval=1, step=1, maxval=4, tooltip=TT_6)

tpType = switch tpTypeInt
    1  => 'None'
    2  => '𝗧.ᴠᴏʟᴀᴛʏ'
    3  => 'R:R'
    4  => 'HH / LL'`

**Comment by RealWenzhangGe** [comment]
> I'm checking and will fix asap

Hi @akumidv ,

Just a heads up: I've already analyzed the issue and submitted a fix in PR #308 (Fix: Repair broken dropdown parameter functionality) :).

The issue is caused by TradingView changing the dropdown structure (from a simple div container to a button[role="combobox"] inside). My PR updates the selectors and click targets in both getStrategyParams and _setStrategyParamsLegacy to handle this new structure correctly.

Feel free to review and merge it when you have time!

**Comment by akumidv** [comment]
I've updated code in repo and have publushed as version. Please check
